src/java/yaw/engine/collision/Collision.java

```diff
@@ -19,8 +20,8 @@
      * @param item2
      * @return isInCollision
      */
-    public static boolean isInCollision(Item item1, Item item2) {
-        if (item1.getBoundingBox() == null && item2.getBoundingBox() == null) {
+    /*public static boolean isInCollision(ItemObject item1, ItemObject item2) {
+        if (item1.gethi() == null && item2.getBoundingBox() == null) {
```

Comment:
what is `gethi()?`  ... and `getBoundingBox()` ... should be `getHitBox()` but what about an item associated to several hitboxes ?  (the test must probably reference the hitboxes, not the itemobjects)